### PR TITLE
(Non-Modular) Fixes a bug where station podpersons aren't friendly to bees

### PR DESCRIPTION
**Pull request by DerFlammenwerfer**
## About The Pull Request
There are two distinct species of Podperson, the primal one that is found in seed vaults, and the station-playable one with less than half the heal rate of the former. Both are intended to be treated as always friendly by bees, but due to an unreported code oversight, the species ID for station-side podpeople was never added to the proc that decides if someone is friendly to bees.

## Changelog

:cl:
fix: Station-side podpeople will no longer be attacked by bees as though they aren't plant people.
/:cl:

### CODE REVIEW

File: code/modules/hydroponics/beekeeping/beebox.dm [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 
 /mob/living/carbon/human/bee_friendly()
-	if(dna && dna.species && dna.species.id == SPECIES_PODPERSON) //bees pollinate plants, duh.
+	if(dna && dna.species && dna.species.id == (SPECIES_PODPERSON || SPECIES_PODPERSON_WEAK)) //bees pollinate plants, duh.
```
Comment (BurgerLUA) [inline code/modules/hydroponics/beekeeping/beebox.dm:19]: This code will not work.

The proper code should be
``if(dna && dna.species && (dna.species.id == SPECIES_PODPERSON || dna.species.id == SPECIES_PODPERSON_WEAK))``

**Review by BurgerLUA — CHANGES_REQUESTED** [review]

File: code/modules/hydroponics/beekeeping/beebox.dm (hunk 1 above)
Comment (DerFlammenwerfer) [reply code/modules/hydroponics/beekeeping/beebox.dm:19]: I'll make this change

**Review by BurgerLUA — APPROVED** [review]

**Comment by Cyprex** [comment]
PR upstream to skyrat or tg, whichever adds primal podpeople

**Comment by nevimer** [comment]
lost to the great rebase...

**Pull request closed without merging**